### PR TITLE
Added Docker Setup Option

### PR DESCRIPTION
**Pull request by kristinyim**
Instructions for setup is at docker-dev.txt
TODO: Add these instructions to readthedocs

### CODE REVIEW

**Comment by kristinyim** [comment]
@felixzhuologist could you take another look?

File: build/semesterly-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:bionic
+
+ADD ./requirements_base.txt /tmp
+
+RUN apt-get install -y \
```
Comment (felixzhuologist) [inline build/semesterly-base/Dockerfile]: does this work? it looks like install and update are mixed up and i would expect to need to use && to run multiple commands. what would make the most sense to me would be to have a different run command/layer for each separate type of thing you're installing (debian package, node, python requirements):
```
RUN apt-get update -y && apt-get install ...
RUN curl .. && apt-get install -y nodejs
RUN pip install -r ...
```

Comment (kristinyim) [reply build/semesterly-base/Dockerfile]: sorry, i didn't push the right version - its up now with &&

Comment (felixzhuologist) [reply build/semesterly-base/Dockerfile]: last nit: can you separate the node and python commands into a separate RUN instruction?

Comment (kristinyim) [reply build/semesterly-base/Dockerfile]: I think I did what you wanted - you're talking about the last 3 lines right?

Comment (felixzhuologist) [reply build/semesterly-base/Dockerfile]: yeah lgtm, thanks!

**Pull request merged**